conf/messages.cy

```diff
@@ -469,32 +469,63 @@ pdf.ruling.aboutDecision.text=Rwyf wedi gwneud y penderfyniad hwn o dan adran 24
 pdf.ruling.samples=Samplau
 pdf.ruling.keepSamples=Mae’ch samplau wedi’u cadw gan y swyddfa hon.
 pdf.ruling.collectSamples=Bydd samplau sydd heb eu casglu neu sydd heb stampiau i’w dychwelyd drwy’r post yn cael eu dinistrio ar ôl 60 diwrnod i ddyddiad yr e-bost am eich dyfarniad. Os hoffech gasglu’ch samplau (neu drefnu i gludwr eu casglu), e-bostiwch <strong>{0}</strong>, gan roi 48 awr o rybudd.
-pdf.ruling.collectSamples.quote=Mae’n rhaid i chi roi cyfeirnod ATaR {0} a nodi faint o’r gloch y bydd y samplau’n cael eu casglu.
 pdf.ruling.coverLetter.title=Llythyr eglurhaol am y Dyfarniad Tariffau Uwch
+
 pdf.ruling.section.ifYouDisagree.title=Yr hyn i’w wneud os ydych yn anghytuno â’r penderfyniad hwn
 pdf.ruling.section.ifYouDisagree=Os ydych yn anghytuno â’r penderfyniad hwn, gallwch naill ai:
 pdf.ruling.section.ifYouDisagree.1=gofyn am adolygiad gan CThEF
 pdf.ruling.section.ifYouDisagree.2=apelio ar dribiwnlys annibynnol
-pdf.ruling.section.ifYouDisagree.cannot=Ni allwch ofyn i CThEF gynnal adolygiad ac apelio ar dribiwnlys annibynnol ar yr un pryd. Fodd bynnag, gallwch apelio ar dribiwnlys annibynnol os ydych yn anghytuno â chanlyniad eich adolygiad.
-pdf.ruling.section.ifYouDisagree.moreInfo=I gael rhagor o wybodaeth am adolygiadau ac apeliadau, gallwch
-pdf.ruling.section.ifYouDisagree.read=ddarllen dogfen HMRC1
-pdf.ruling.section.ifYouDisagree.phone=neu ffonio 0300 200 3705.
+pdf.ruling.section.ifYouDisagree.within=Mae'n rhaid i chi wneud hynny cyn pen 30 diwrnod o ddyddiad y llythyr hwn.
```

Comment:
Apostrophe is not displayed currently. Please use the curved one for consistency in welsh. Update to `Mae’n`, thanks